assets/stylesheets/common/common.scss

```diff
@@ -237,7 +237,7 @@ body.composer-open .topic-chat-float-container {
 
 .topic-chat-drawer-content {
   box-sizing: border-box;
-  height: 100%;
+  height: calc(100% - #{$header-height});
```

Comment:
I think `var(--header-offset)` might be the more modern version of this?

Reply:
> header-offset

no in this case $header-height is a local variable of the file, related to height of the drawer's header and not the discourse core header.

Reply:
Ohhhh I see, sounds good :+1:

Reply:
https://github.com/discourse/discourse-chat/blob/47ceb26434cd00e35c34aca564360083afe7ba8a/assets/stylesheets/common/common.scss#L1